fastly/data_source_fastly_tls_configuration_ids_test.go

```diff
@@ -0,0 +1,21 @@
+package fastly
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"testing"
+)
+
+func TestAccFastlyDataSourceTLSConfigurationIDs(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFastlyAccDataSourceTLSConfigurationIDs,
+				Check:  resource.TestCheckResourceAttrSet("data.fastly_tls_configuration_ids.subject", "ids.#"),
+			},
+		},
+	})
+}
+
+const testAccFastlyAccDataSourceTLSConfigurationIDs = `data "fastly_tls_configuration_ids" "subject" {}`
```

Comment:
I think this is underneath because it's a test piece of HCL config and they seem to normally be at the bottom of the files when they are functions, so would make sense to group them even if some of them don't have any variable data. Happy to shift it around if you'd prefer.

Reply:
No that's fine. I didn't really pay attention to the fact it was mock hcl, and I agree that those always appear to be at the bottom of the test file 👍🏻